### PR TITLE
line editing for odoc.mld

### PR DESCRIPTION
**Pull request by christinerose**
Initial suggestions to improve the `odoc` pages and make them consistent with the style guide.

### CODE REVIEW

File: doc/odoc.mld [hunk 1]
```diff
@@ -23,7 +23,7 @@ val float_dsig : int -> float t
     and floats. *)
 ]}
 
-These comments are picked up by [odoc] and {{!Fmt.float_dsig}turned into html}, or TeX or manpages.
+These comments are picked up by [odoc] and {{!Fmt.float_dsig}turned into HTML}, TeX, or manpages.
```
Comment (christinerose) [inline doc/odoc.mld:26]: Should this be TeX or LaTeX?

Comment (Julow) [reply doc/odoc.mld:26]: LaTeX !

Comment (christinerose) [reply doc/odoc.mld:26]: Glad I asked! I thought TeX was a little odd.

File: doc/odoc.mld [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+from your source files and your [.mld] files, then outputs HTML, LaTeX and
 man pages. The pages you are reading now are rendered using [odoc].
 
 Text inside doc comments (delimited by [(** ... *)]) is marked up in
 [odoc] syntax:
 
 {[
 val float_dsig : int -> float t
-(** [float_dsig d] rounds the normalized {e decimal} significand
+(** [float_dsig d] rounds the normalised {e decimal} significant
```
Comment (panglesd) [inline doc/odoc.mld:16]: I think "significand" is [intended](https://en.wikipedia.org/wiki/Significand).

Also, this part is actually a quote: https://ocaml.org/p/fmt/0.8.3/doc/Fmt/index.html#val-float_dsig
So I don't think we should modify it?

Comment (christinerose) [reply doc/odoc.mld:16]: Oh good to know! Thank you!

**Comment by EmileTrotignon** [comment]
I think that's good to be merged.

**Review by EmileTrotignon — APPROVED** [review]
I think that's good to be merged

**Pull request merged**